code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -0,0 +1,976 @@
+/*
+Tiers and Requirements
+
+Pieces of scripture require certain follower counts, contruction value, and active caches in order to recite.
+Drivers: Unlocked by default
+Scripts: 5 servants and a cache
+Applications: 8 servants, 3 caches, and 50 CV
+Revenant: 10 servants and 100 CV
+Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
+*/
+
+/datum/clockwork_scripture
+	var/name = "scripture"
+	var/desc = "Ancient Ratvarian lore. This piece seems particularly mundane."
+	var/list/invocations = list() //Spoken over time in the ancient language of Ratvar. See clock_unsorted.dm for more details on the language and how to make it.
+	var/channel_time = 10 //In seconds, how long a ritual takes to chant
+	var/list/required_components = list("belligerent_eye" = 0, "vanguard_cogwheel" = 0, "guvax_capacitor" = 0, "replicant_alloy" = 0, "hierophant_ansible" = 0) //Components required
```

Comment:
As far as I'm concerned, there's no difference between strings and defines here - I'd argue that strings make varediting more clear.